lib/ical/parse.js

```diff
@@ -232,9 +232,20 @@ ICAL.parse = (function() {
     var multiValue = false;
     var structuredValue = false;
     var propertyDetails;
+    var splitName;
+    var ungroupedName;
+
+    // fetch the ungrouped part of the name
+    if (name.indexOf('.') !== -1) {
```

Comment:
Likewise to comment below, we need to differentiate iCalendar vs vCard here by using the designSet.